core/common/src/BufferExt.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.io
+
+import kotlinx.io.bytestring.ByteString
+import kotlinx.io.bytestring.buildByteString
+
```

Comment:
Missed it during the original review.
We typically prefer extensions for `class Foo` either placed in the same file (-> class`FooKt`) for small and compact additions, or in plural form of the entity -- `Foos` (`Channels.kt`, `Strings.kt`, `Serializers.kt`). Bit nicer stacktraces, a bit more consistent

Reply:
We also use `@file:JvmMultifileClass` for files spread across platforms, but maybe it's a bit too redundant

Reply:
Thanks for fixing it!